recipes/eigen/run_test.bat

```diff
@@ -0,0 +1,3 @@
+g++ -I${PREFIX}/include -o test test.cc
```

Comment:
So, as MinGW is around I tried to use `g++`, but that doesn't seem to work for me. Is there some configuration AppVeyor needs to do this?